harvest/api/dummy.py

```diff
@@ -90,7 +108,7 @@ def fetch_price_history(self,
         elif unit == 'DAY':
             interval = dt.timedelta(days=int(value))
         else:
-            print('Error')
+            error(f'Interval unit was {unit} not MIN, HR, or DAY!')
```

Comment:
replace `print` with the appropriate `logger` level message. See: https://docs.python.org/3/library/logging.html